ingestion-beam/src/test/java/com/mozilla/telemetry/DecoderMainTest.java

```diff
@@ -20,25 +27,39 @@
   @Rule
   public TemporaryFolder outputFolder = new TemporaryFolder();
 
-  private String outputPath;
-
-  @Before
-  public void initialize() {
-    outputPath = outputFolder.getRoot().getAbsolutePath();
-  }
+  @Rule
+  public final RedisServer redis = new RedisServer();
```

Comment:
Since RedisServer is now needed in two places, I implemented a junit `Rule` for it that gets used here and in DecoderTest